style.css

```diff
@@ -98,4 +103,273 @@ li {
   cursor: pointer;
 }
 
+.vertical-space {
+  height: 50px;
+}
+
 /* hero section ends */
+
+/* works section starts */
+.works {
+  margin-top: 50px;
+  background-color: #f7f7f9;
+}
+
+.grid-container {
```

Comment:
- [ ] There is more white spaces(empty spaces) around your text content in each project card which is not present in the Figma design. Take a look at the comparison below 👇 
**Figma design**
![image](https://user-images.githubusercontent.com/90222110/177011415-3fdc675b-92d7-46fd-84dc-4e41917e9932.png)
**Your portfolio design**
![image](https://user-images.githubusercontent.com/90222110/177011491-3ad7d9da-c122-4d66-a8ad-41775c445ec9.png)
As a developer, you should pay attention to all details because it is what makes you outstanding 😃. Please correct that spacing.